_field-types/mapping-parameters/doc-values.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: doc_values
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 25
+has_children: false
+has_toc: false
+---
+
+# Doc_values
+
+OpenSearch indexes most fields by default for searching. While the inverted index allows quick term-to-document lookups, doc values enable efficient document-to-term access for sorting, aggregations, and scripting.
```

Comment:
"By default, OpenSearch indexes most fields so that they can be searched"? What inverted index (hasn't been mentioned)? Should "doc values" be "the `doc_values` parameter"? Should "access" be "lookups" (as in the first instance)?

Reply:
@bowenlan-amzn Will you provide insight to the above questions? I have little to no guidance on the feature and need developer input. Thank you.

Reply:
doc values is fine to me.
access can be changed to lookup.

Reply:
@natebower We've addressed editorial feedback. This PR is ready for re-review. Thanks!